servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ParentConnectionContext.java

```diff
@@ -238,19 +244,22 @@ public final void handlerRemoved(ChannelHandlerContext ctx) {
         private void doChannelClosed(final String method) {
             parentContext.notifyOnClosingImpl();
 
-            if (hasSubscriber()) {
-                tryFailSubscriber(StacklessClosedChannelException.newInstance(H2ParentConnectionContext.class, method));
-            }
+            tryFailSubscriber(StacklessClosedChannelException.newInstance(H2ParentConnectionContext.class, method));
```

Comment:
since this method now returns a boolean, should the code do something with it here?

Reply:
No, it's just best effort to propagate an exception. No need to close the channel bcz this event means that the channel is already closed